src/main/java/org/mozilla/jss/netscape/security/x509/IPAddressName.java

```diff
@@ -209,8 +209,7 @@ public static int fillIPv4Address(String s, byte[] address, int start) {
         try {
             int end = start + nt;
             for (int i = start; i < end; i++) {
-                Integer j = new Integer(Integer.valueOf(st.nextToken()));
-                address[i] = (byte) j.intValue();
+                address[i] = Byte.valueOf(st.nextToken());
```

Comment:
Here we're parsing IPv4 values in the range of `0..255`, but Java's `byte` range is `-128..127`. I suspect the original code was trying to avoid a `NumberFormatException`, so it's parsing the token into an `Integer` first, then downcast it into `byte`. We might be able to do this instead:
```
// parse token into Integer then downcast into byte to avoid NumberFormatException
address[i] = (byte) Integer.parseInt(st.nextToken());
```
Should we do this change separately since it requires further inspection?
Maybe we can add a unit test for this too.

Btw, I think we should use `parseInt/parseByte` instead of `valueOf` if the result that we want is a primitive.